csp_billing_adapter_local/plugin.py

```diff
@@ -111,3 +114,82 @@ def save_csp_config(
 ):
     """Save specified content as local storage csp_config contents."""
     update_csp_config(config, csp_config, replace=True)
+
+
+@csp_billing_adapter.hookimpl(trylast=True)
+def get_usage_data(config: Config):
+    """
+    Retrieves the current usage report from the application API
+
+    :param config: The application configuration dictionary
+    :return: Return a dict with the current usage report
+    """
+    usage_data = _make_request(config.get('api'))
+    usage_data_list_name = ''.join(list(usage_data.keys()))
```

Comment:
I didn't notice this originally. If there are any unexpected keys this will lead to a key error in line 139.

Reply:
Good catch